library/built-in/_prefabs/prefab_latent_v3.ts

```diff
@@ -33,11 +46,18 @@ export const run_latent_v3 = async (p: { opts: OutputFor<typeof ui_latent_v3>; v
 
     // case 1. start form image
     if (opts.image) {
-        const _img = run.loadImage(opts.image.imageID)
+        const _img = run.loadImage(opts.image.image.imageID)
```

Comment:
hah ! forgot to remove this line. it's not needed anymore, loadImage is some legacy api

the line below `        const image = await _img.loadInWorkflow()` already do everything we need.
if we want to access the width / height of the image, we can simply do `opts.image.width`